### PR TITLE
Add view_activity permissions

### PR DESCRIPTION
**Pull request by servingUpAces**
To ensure that only members of the service have access to pages like the dashboard, view activity, view team member, the user_has_permissions decorator was added to the pages. However, a user with no permissions on the service (thought of as default permission to view only access) would not have permission to view the page. The platform admin user also was unable to see the dashboard page for all the services. This pull request add a new permission, view_activity. All users will get this permission as a default. The pull request on notifications-api to update the data should be merge before this one, as it will update all existing users with the new permission. 

The access_developer_docs was also removed as it is not a required permission. Everyone can see the documentation.

### CODE REVIEW

File: app/main/views/manage_users.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def invite_user(service_id):
             email_address,
             ','.join(
                 role for role in roles.keys() if request.form.get(role) == 'y'
-            )
+            ).join('view_activity')
```
Comment (mannickutd) [inline app/main/views/manage_users.py]: Probably should put a comment here as to why we are joining this 'view_activity' string to the list.

File: app/main/views/manage_users.py [hunk 2]
```diff
@@ -98,7 +98,7 @@ def edit_user_permissions(service_id, user_id):
             user_id, service_id,
             permissions=set(chain.from_iterable(
                 permissions for role, permissions in roles.items() if form[role].data
-            ))
+            )) | {'view_activity'}
```
Comment (mannickutd) [inline app/main/views/manage_users.py]: Another comment here as to what 'view_activity' is.

File: environment_test.sh [hunk 3]
```diff
@@ -1,6 +1,6 @@
 export NOTIFY_ADMIN_ENVIRONMENT='config.Test'
 export ADMIN_CLIENT_SECRET='dev-notify-secret-key'
 export ADMIN_CLIENT_USER_NAME='dev-notify-admin'
-export API_HOST_NAME='http://localhost:6011'
+export API_HOST_NAME='http://localhost:6311'
```
Comment (mannickutd) [inline environment_test.sh]: Why is this changed?

File: tests/app/main/views/test_manage_users.py [hunk 4]
```diff
@@ -305,45 +274,46 @@ def test_no_permission_manage_users_page(app_,
...
+                    user_id=active_user_with_permissions.id))
             assert response.status_code == 302
             assert response.location == url_for(
-                'main.manage_users', service_id=service['id'], _external=True)
+                'main.manage_users', service_id=service_one['id'], _external=True)
+            mock_remove_user_from_service.assert_called_once_with(service_one['id'],
+                                                                  str(active_user_with_permissions.id))
+
+
+def _login_user(user, client, mocker, service):
```
Comment (mannickutd) [inline tests/app/main/views/test_manage_users.py]: Why is there another way to login now?
Why not just adjust the original way to login with this new way?
Now we have 2 ways to login a test user.

File: app/main/views/manage_users.py (hunk 1 above)
Comment (quis) [inline app/main/views/manage_users.py]: I don’t think this is doing what you think it’s doing:
```
In [6]: ",".join(['one', 'two', 'three']).join('four')
Out[6]: 'fone,two,threeoone,two,threeuone,two,threer'
```

Comment (quis) [inline app/main/views/manage_users.py]:
```python
','.join(list(role for role in roles.keys() if request.form.get(role) == 'y') + 'view_activity')
```

Would do it.

File: environment_test.sh (hunk 3 above)
Comment (servingUpAces) [inline environment_test.sh]: There has been a few times where the test pass when there are missing mocks, so set it to something other than the port we are using.

Comment (quis) [inline environment_test.sh]: This is a good idea.

**Comment by mannickutd** [comment]
:+1:
